scheduler/src/cook/passport.clj

```diff
@@ -0,0 +1,15 @@
+(ns cook.passport
+  (:require [clojure.data.json :as json]
+            [clojure.tools.logging :as log]
+            [congestion.limits :refer [RateLimit]]
+            [cook.config :as config]
+            [cook.rest.impersonation :refer [impersonation-authorized-wrapper]]
+            [plumbing.core :refer [fnk]]))
+
+(defn log-passport-event
```

Comment:
How about `stamp` to go along with the passport analogy? Callers would then be calling it with `(passport/stamp {...})`, which seems both clear and concise.

Reply:
Yeah, I like that

Reply:
Still planning to do this?

Reply:
Hmmm, I don't know if passport/stamp is clear enough

Reply:
my vote is for `passport/log-event`

Reply:
I'm fine with `passport/log-event`

Reply:
Sorry, I should have included a note when I posted the PR. Ultimately decided not to change it b/c I think that `passport/stamp` isn't quite clear enough for people who aren't already very familiar with the passport analogy. I'm good with changing it to `passport/log-event` though to shorten it up and avoid the repetition of `passport`.